src/libraries/Microsoft.PowerFx.Connectors/Public/ConnectorRelationship.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using Microsoft.PowerFx.Core.Utils;
+
+namespace Microsoft.PowerFx.Connectors
+{
+    public class ConnectorRelationship
```

Comment:
>ConnectorRelationship [](http://example.com/codeflow?start=17&length=21)

Is this CDP specific? 

If so, how do non-CDP shims (ServiceNow) leverage it?

Reply:
for now, yes
as discussed this is a temporary thing
I'll add [Obsolete] to it

Reply:
I added [Obsolete]